### DISCUSSION TITLE
haven't received students benefits even after approval.

### DISCUSSION THREAD

**Discussion by bytewickx** [community/community#133531] [GitHub Education] [answered]
### Hi everyone,

![Screenshot 2024-07-25 190921](https://github.com/user-attachments/assets/070f6c72-ec3c-42ae-868d-dd8d0c192b44)

**Comment by davevad93** [comment, marked as answer]
Hi @tacticiancoder ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @tacticiancoder was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470

**Reply by bytewickx** [reply]
Received it on the 6th day, thanks for the help much appreciated.
